src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -124,6 +142,7 @@ public IReadOnlyList<Token> Tokenize(string expressionText)
         /// <returns></returns>
         public ParseResult Parse(string expressionText, ParserOptions options = null)
         {
+            options ??= this.GetDefaultParserOptionsCopy();
```

Comment:
>this [](http://example.com/codeflow?start=24&length=4)

no need

Reply:
If (options == null) { 
  options = this.GetDefaultParserOptionsCopy();
}
We don't want to pass null into line 146.